exports/@material-ui/core/4.10.1.json

```diff
@@ -113,5 +113,9 @@
   "./Tooltip": "./esm/Tooltip/Tooltip.js",
   "./Typography": "./esm/Typography/Typography.js",
   "./Unstable_TrapFocus": "./esm/Unstable_TrapFocus/Unstable_TrapFocus.js",
-  "./Zoom": "./esm/Zoom/Zoom.js"
+  "./Zoom": "./esm/Zoom/Zoom.js",
+  "./styles/*": "./styles/*.js",
```

Comment:
are you sure about this? I don't think "./styles/*" is meant to be imported. In the documentation I only see:

```
import { styled } from '@material-ui/core/styles';
```

which would mean this line should be:

```
"./styles": "./esm/styles/index.js"
```

see https://material-ui.com/styles/basics/